Gather homepage

dsusviela opened the pull request:
New:
 - Podfile under ./ios 
 - Installed mapbox with `npm install @mapbox/react-native-mapbox-gl --save`

Modified:
- android(aka project):build.gradle:
  - added jitpack repo
- app: build.gradle:
  - added mapbpx dependency
  - changed targetSdkVersion from 22 to 26
- settings.gradle
  - added mapbox to projects
 - MainApplication.java
   - added several functions and parameters missing from the example here [https://github.com/mapbox/react-native-mapbox-gl/blob/master/android/install.md](url)
 - Gather.js
   - Added the map from the example in [https://www.mapbox.com/help/first-steps-react-native-sdk/](url)

Please note that several extra files may have been modified during the installation of mapbox.

frabago commented on src/components/Gather/Gather.js:
```diff
@@ -1,49 +1,27 @@
+            styleURL={Mapbox.StyleURL.Street}
+            zoomLevel={15}
+            centerCoordinate={[11.256, 43.770]}
+            style={styles.container}>
```
If you delete the styles import  when the user go to the gather page the app display an error saying that doesn't find the styles variable. 

Fixing that import when I go to the gather page I only see a green screen without the map.

dsusviela replied:
Should be fixed in next commit, its about to get pushed

frabago commented on src/components/Gather/Gather.js:
```diff
@@ -1,49 +1,27 @@
-import strings from '../../localization';
-import getUser from '../../selectors/UserSelector';
-import getRole from '../../selectors/RoleSelector';
-import styles from './styles';
```
Add again the import

frabago commented on src/components/Gather/Gather.js:
```diff
@@ -1,49 +1,27 @@
-const mapStateToProps = state => ({
-  user: getUser(state),
-  role: getRole(state),
+const stylesGather = StyleSheet.create({
```
It should be in the style file

frabago commented on src/components/Gather/Gather.js:
```diff
@@ -1,49 +1,27 @@
-  };
+Mapbox.setAccessToken('pk.eyJ1IjoicXFtZWxvIiwiYSI6ImNqbWlhOXh2eDAwMHMzcm1tNW1veDNmODYifQ.vOmFAXiikWFJKh3DpmsPDA');
 
+export default class App extends Component {
```
Seem to be that you are not running eslint and prettier

dsusviela commented:
Gather.js
  - Fixed several errors with eslint and prettier
  - Removed styles from the js file
  - Added styles import wrongfully removed

Gather/styles.js
  - Fixed several errors with eslint and prettier
  - Changed StyleSheet created name from styles to stylesGather to avoid future conflicts
  - Added a mapContainer as it differs from normal containers
